ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.test.js

```diff
@@ -795,6 +795,46 @@ describe('Confirm Transaction Base', () => {
         });
       });
 
+      describe('when determines the recipient from transaction data args', () => {
```

Comment:
Do we need a negative test to verify that a non-transfer function matching the ABI doesn't render the param?

Reply:
Done.